vizro-core/docs/pages/user_guides/dashboard.md

```diff
@@ -184,4 +185,66 @@ To create a dashboard, do the following steps:
             Vizro().build(dashboard).run()
         ```
 
+
+## Adding a dashboard title
```

Comment:
If we add a section here, we should also add a section for the top bit, otherwise it looks like there is only an exaplnation on the title. Maybe `## Dashboard configuration options`

Reply:
Renamed to ## Using dashboard configuration options